code/modules/client/client_procs.dm

```diff
@@ -598,6 +600,9 @@
 			var/err = query_insert.ErrorMsg()
 			log_game("SQL ERROR during log_client_to_db (insert). Error : \[[err]\]\n")
 			message_admins("SQL ERROR during log_client_to_db (insert). Error : \[[err]\]\n")
+		// This is their first connection instance, so TRUE here to nofiy admins
+		// This needs to happen here to ensure they actually have a row to update
+		INVOKE_ASYNC(src, /client/.proc/get_byond_account_date, TRUE) // Async to avoid other procs in the client chain being delayed by a web request
```

Comment:
Long term, the TRUE here might be a good idea to get notified about new players.
However:
- in the short term, this is gonna spam admins a LOT as each player connects after this PR is merged, which is bad, as it trains admins to ignore the notice
- take it from me (and my experience with ipintel and similar warnings).... the more warnings you give admins, the greater the chances that they ignore the warnings entirely. Instead of doing an alert to admins every time a new player connects, you should instead only notify admins if its a <3d old player connecting for the first time. Limit the number of warnings to a bare minimum to increase their signal-to-noise ratio.

Reply:
This is only for players who have **never** been seen by the game before (IE: They have no row in the `player` table)